### PR TITLE
[WIP] Blueprint tests

### LINKED ISSUE #101: Set up tests
Eventually I'd like to have everything tested with multiple versions of ember-cli, but I think covering these two is a good start:
- [x] Test that the built in ember tests run with coffee-files
- [x] Test all the blueprints with the [`ember-cli-blueprint-test-helpers`](https://github.com/ember-cli/ember-cli-blueprint-test-helpers)

To do this, the project will first need to be upgraded to the newest ember-cli `ember addon` blueprints.
- [x] Upgrade the project to the newest `ember addon` blueprints

### PR DESCRIPTION
**Pull request by kimroen**
Fixes #101

### CODE REVIEW

**Comment by kimroen** [comment]
Added some tests where one of them should fail just to check that everything is all right, but then they both passed.

**Comment by trabus** [comment]
You're using the wrong property to check the file, it should be `contains`, not `contents`:

> files (array): Array of files to assert, represented by objects with file, exists, contains, or doesNotContain properties. Example object:
```
files: [
  {
    file: 'path-to-file.js', 
    contains: ['file contents to compare'], 
    doesNotContain: ['file contents that shouldn\'t be present'], 
    exists: true //default true
  }
]```

File: node-tests/blueprints/adapter-test.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+      ]
+    });
+  });
+
+  it('adapter foo', function() {
+    return generateAndDestroy(['adapter', 'foo'], {
+      files: [
+        {
+          file: 'app/adapters/foo.coffee',
+          contents: [
```
Comment (trabus) [inline node-tests/blueprints/adapter-test.js]: Should be `contains` here

**Comment by kimroen** [comment]
I came back to take a look at this today, but so much had happened with the blueprint test helpers that I figured it would be better to start over. I did that over here: #129

Thanks for the help!

**Pull request closed without merging**